src/main/java/com/google/step/datamanager/RestaurantPlaceManager.java

```diff
@@ -0,0 +1,10 @@
+package com.google.step.datamanager;
+
+import java.util.List;
+import java.util.Set;
+
+public interface RestaurantPlaceManager {
+  public void updatePlacesOfRestaurants(long restaurantId, List<String> placeIds);
+
+  public Set<String> getPlaceIdsOfRestaurant(long id);
```

Comment:
```suggestion
  public Set<String> getPlaceIdsOfRestaurant(long restaurantId);
```

is clearer and disambiguates between the two IDs being referred to